src/DynamoCore/Search/SearchLibrary.cs

```diff
@@ -45,7 +45,9 @@ internal void Update(TEntry entry, bool isCategoryChanged = false)
                 keys.Add(entry.Description.ToLower(), 0.1);
 
                 foreach (var tag in entry.SearchTags.Select(x => x.ToLower()))
+                {
                     keys.Add(tag, 0.5);
```

Comment:
btw - I have no idea why this is .5 - shouldn't it be the actual weight?

Reply:
I guess search tag is not as identical as `Name`